cast-dssspell.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$(seth chain --rpc-url="$ETH_RPC_URL")" == "goerli" ]] || { echo "Please set a Goerli ETH_RPC_URL"; exit 1; }
+
+CHANGELOG=0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F
+MCD_ADM=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_ADM")")")
+MCD_GOV=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_GOV")")")
+MCD_IOU=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_IOU")")")
+MKR=$(seth --to-wei 10001 ether)
```

Comment:
This is still only `10,001` MKR.  We need this to be `100,001` MKR.
```suggestion
MKR=$(seth --to-wei 100001 ether)
```
```suggestion
MKR=$(seth --to-wei 10001 ether)
```

Reply:
just checked with @gbalabasquer in chat.
the deployer balance is currently at 98,999 MKR.
the `deposits` in DSChief for the deployer is 100,001 MKR.
As discussed with Gonza, it seems we should free 1 MKR first and then consider adding some logic to make sure the `deposits` stays in 100K MKR range and `lock` whatever is necessary to pass the `hat` by 1 wei.

Reply:
attempted to generalize the script in 5fe7ad8